res/skins/Deere/equalizer_rack_parameter_right.xml

```diff
@@ -19,6 +19,7 @@
     <Children>
 
       <SizeAwareStack>
+        <Size>40f,-1me</Size>
```

Comment:
Hm, neither did I ;) it was late and I just reversed elements' order in ...left.xml and diffed